tests/ga/test_cgroupapi.py

```diff
@@ -301,7 +301,7 @@ def mock_popen(command, *args, **kwargs):
                         shell=True,
                         timeout=300,
                         cwd=self.tmp_dir,
-                        env={},
+                        env={}.update(os.environ),
```

Comment:
These tests were failing on the new runs for 3.4. The reason is that they have a dependency on the "python3" executable;  on Github actions we were installing Python 3 even on runs that were using our 3.4 venv, so there was a "python3" in the system PATH and the tests were able to run. Now, we no longer install Python 3 for the 3.4 test runs, and "python3" is only in the PATH for the venv. These tests need to inherit the environment from the test runner (nosetests) in order to have the correct PATH. The Agent does something similar (adding os.environ) when invoking extensions.